clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/MigrateServerGroupStrategy.java

```diff
@@ -159,6 +159,7 @@ public synchronized MigrateServerGroupResult generateResults(ServerGroupLocation
       deployDescription.setKeyPair(keyPair != null ? keyPair : launchConfig.getKeyName());
       deployDescription.setAssociatePublicIpAddress(launchConfig.getAssociatePublicIpAddress());
       deployDescription.setCooldown(sourceGroup.getDefaultCooldown());
+      deployDescription.setGroupMetrics(sourceGroup.getGroupMetrics());
```

Comment:
Travis is complaining about this line - might be safe to omit (I don't think anyone outside Netflix ever used the migration features)